### DISCUSSION TITLE
Openlayers examples of integration for frameworks such as Next.js, Remix etc?

### DISCUSSION THREAD

**Discussion by Seefer** [openlayers/openlayers#15558] [General]
I dabbled getting Openlayers working in Next.js but had problems figuring out how to make it play nicely with Tailwind.css. I had to resort to using inline styles on the map container i order for the map top render. The Openlayers examples @ https://openlayers.org/en/latest/examples/ would benefit from code showing integration with popular frameworks, including CSS frameworks.

**Comment by heronlancellot** [comment]
hey @Seefer  ! Agree  ! 
I only found that one with Next JS + Tailwind CSS + React 
https://github.com/OthyTenk/openlayers-nextjs-map check this out : )
